api/_mixins/DocumentOrShadowRoot__ShadowRoot.json

```diff
@@ -325,6 +325,288 @@
           }
         }
       },
+      "getAnimations": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/ShadowRoot/getAnimations",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "84"
+              },
+              {
+                "version_added": "83",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "#enable-experimental-web-platform-features"
+                  }
+                ]
+              },
+              {
+                "version_added": "67",
+                "version_removed": "83",
+                "partial_implementation": true,
+                "notes": "Implements an older version of the specification and not supported on <code>ShadowRoot</code>, see <a href='https://crbug.com/828585'>Chromium bug 828585</a>.",
```

Comment:
Any mention of `ShadowRoot` in the notes here will need attention. This one suggests the entry should simply not be here.